run_node.sh

```diff
@@ -43,6 +43,12 @@ $0 --name my-aleph-node --dns some.domain --version r-12.1
 EOF
 }
 
+prepare_directories() {
+    mkdir -p "${HOST_BASE_PATH}"
```

Comment:
Why not `CHAIN_DATA_DIR=${HOST_BASE_PATH}/${CHAIN_DATA_DIR}; mkdir -p "${CHAIN_DATA_DIR}"`?